docs/src/pages/guides/new-notifications.md

```diff
@@ -0,0 +1,173 @@
+---
+id: new-notifications
+title: New Notifications
+---
+
+## New Notifications
+
+Out of the box, Monika support a multitude of notification channels, from chat applications such as Slack, Discourse, Microsoft Teams, Google Chats to email based platform such as Mailgun, to generic webhooks and the SMTP protocol ([click here for the full list](https://monika.hyperjump.tech/guides/notifications)). Should you have a favorite tool not already supported, we invite you to add it to Monika's growing list. Here's how.
+
+## Prerequisites
+
+Adding a custom notification to Monika is quite straight forward. There are just a few requirements.
+
+1. Some basci javascript may be helpful.
+2. Some familiarity with git is required to clone and create pull requests.
+3. A Github account to fork and clone the [Monika open source repository](https://github.com/hyperjumptech/monika).
+4. A development environment already set up for javascript/typescript. See the project [README](https://github.com/hyperjumptech/monika/blob/main/README.md) for an overview.
```

Comment:
set up for javascript/typescript -> set up for developing in JavaScript/TypeScript.